Assets/Scripts/Models/Furniture.cs

```diff
@@ -143,6 +151,34 @@ private Furniture(Furniture other)
 
     public event Action<IPowerRelated> PowerValueChanged;
 
+    // PathfindingModifier is added to PathfindingCost to get the final amount
+    public float PathfindingModifier
+    {
+        get
+        {
+            return pathfindingWeight;
+        }
+
+        set
+        {
+            pathfindingWeight = value;
+        }
+    }
```

Comment:
Stylecop is perfectly fine with them being on separate lines. After having to make so many properties multiline on Stylecop Saturday, I just got in the habit of making all Properties like this to start with. But as long as Stylecop doesn't complain about one-lining them, I'll do it.